### ISSUE TITLE
not working: /// <reference types="chrome"/> 

### ISSUE THREAD

**Issue by yigalirani** [biomejs/biome#4224]
### Environment information

```bash
CLI:
  Version:                      1.9.3
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           "v20.15.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "npm/10.7.0"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### Rule name

noUndeclaredVariables

### Playground link

https://biomejs.dev/playground/?lintRules=all&code=LwAvAC8AIAA8AHIAZQBmAGUAcgBlAG4AYwBlACAAdAB5AHAAZQBzAD0AIgBjAGgAcgBvAG0AZQAiAC8APgAKAGMAaAByAG8AbQBlAC4AZABlAHYAdABvAG8AbABzAC4AaQBuAHMAcABlAGMAdABlAGQAVwBpAG4AZABvAHcALgBlAHYAYQBsACgAJwBmAG8AbwAoACkAJwApAA%3D%3D

### Expected result

no warnings because `/// <reference types="chrome"/> ` should declare the variable chrome

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by yigalirani** [comment]
user error. can use the globals in biome.json for this

**Comment by nnennajohn** [comment]
@yigalirani , how did you resolve this? I added chrome to my "globals", and the not found error still persists.

<img width="551" alt="Screenshot 2024-10-11 at 3 19 18 PM" src="https://github.com/user-attachments/assets/d2975ccc-a81b-4016-836c-631c7b1a6b63">
<img width="882" alt="Screenshot 2024-10-11 at 3 19 30 PM" src="https://github.com/user-attachments/assets/f6431092-6b99-48cb-a05d-94a5ec36d613">

**Comment by yigalirani** [comment]
maybe the problem is with your usage of the `extends` command. i dont use it
![image](https://github.com/user-attachments/assets/f0f1c57e-6395-4d29-923a-b6391bdcf5c4)
